src/components/Transaction.tsx

```diff
@@ -1,31 +1,112 @@
 import React, { useEffect, useState } from 'react';
 import { gql } from '../fetch';
-import type { Signature } from '@solana/web3.js';
-import {  } from '@solana/web3.js';
+import type { Signature, Slot, Transaction, TransactionMessage } from '@solana/web3.js';
 
 /**
  * Component GraphQL query.
  */
+
+// Can add more fields based on requirement
 const source = /* GraphQL */ `
+    fragment TransactionMessageAccountKey on TransactionMessageAccountKey {
+        pubkey 
+        signer 
+        source 
+        writable
+    }
```

Comment:
What's this for?

Reply:
Technically there's more to using fragments than just that, but we can save fragments and more GraphQL features for later. Right now let's just lay out the components and pages, and start to tie the app together. Later we can rewire all the queries (using Relay) and optimize this thing.